example/heatEquation2D/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright 2023 Benjamin Worpitz, Jan Stephan
+# SPDX-License-Identifier: ISC
+#
+
+################################################################################
+# Required CMake version.
+
+cmake_minimum_required(VERSION 3.22)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+################################################################################
+# Project.
+
+set(_TARGET_NAME heatEquation2D)
+
+project(${_TARGET_NAME} LANGUAGES CXX)
+
+
+################################################################################
+# PNGwriter
+################################################################################
+
+# find PNGwriter installation
+find_package(PNGwriter 0.7.0 REQUIRED CONFIG)
```

Comment:
if we set this to require this PR will not pass the CI.
Should we make this optional and guard the IO with `#ifdef`?

Reply:
Yes, we can make this optional and gaurd with ifdefs. We can then also treat openPMD the same way as I assume it will have the same problem